explorer/src/pages/Schema/index.tsx

```diff
@@ -40,9 +50,11 @@ export const Schema = () => {
         <div className="flex flex-col gap-2">
           <p className="text-xs text-text-quaternary not-italic font-normal uppercase">{t("schema.context")}</p>
           <a
-            href={schema.context}
+            href={isContextURL ? schema.context : undefined}
```

Comment:
I think that `{schema.context}` shouldn't be a link in case when `isContextURL` is falsy.